api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -828,6 +828,23 @@ private void enableDispensing() {
 
         // ToDo:  Add this back when the widget is changes to show all obs groups (not just one) per encounter
 
+        apps.add(addToClinicianDashboardFirstColumn(app(Apps.NOTES_SUMMARY,
```

Comment:
Actuallyl instead of having to create a whole new component here, let's just include this within the configuration of the existing "COMMENT_FORM" app, see CALF:3167

Reply:
Thanks @mogoodrich ! I have updated the PR and also added the sortOrder config parameter. Thanks!